### ISSUE TITLE
Found Broken Link in "v3vsv4.md" Documentation

### ISSUE THREAD

**Issue by Gearheads** [kubernetes-sigs/kubebuilder#4434]
Hello 😄

I found a link on this https://book.kubebuilder.io/migration/v3vsv4 page of the documentation portal that does not resolve.
https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/migration/v3vsv4.md?plain=1#L67

Screen shot in case it helps:
<img width="1512" alt="Screenshot 2024-12-18 at 3 08 05 PM" src="https://github.com/user-attachments/assets/1a150f43-8d44-43e8-8437-9bece0f0b789" />

I think the link needs to be updated to this one, https://book.kubebuilder.io/plugins/available/go-v4-plugin?

Please let me know if I am mistaken, or if I could create a PR. I would be happy to help create a PR if needed.

**Comment by camilamacedo86** [comment]
It seems fixed for the master branch: https://master.book.kubebuilder.io/
Therefore, the fix will be available in the next release (soon)

See; https://master.book.kubebuilder.io/migration/v3vsv4

So I am closing this one.

**Comment by Gearheads** [comment]
hi @camilamacedo86, apologies if I am misunderstanding something, it looks like this link still returns 404?
https://master.book.kubebuilder.io/plugins/go-v4-plugin.html

Here is the section within https://master.book.kubebuilder.io/migration/v3vsv4 that I am referring to:
> TL;DR of the New `go/v4` Plugin
> Further details can be found in the [go/v4 plugin section](https://master.book.kubebuilder.io/plugins/go-v4-plugin.html)
